lib/weather.js

```diff
@@ -136,14 +134,15 @@ function parseWeatherXml(xml, callback) {
                 var pop = forecast.abbreviatedForecast[0].pop[0]._;
                 weather.forecasts.push({
                     period: forecast.period[0]._,
-                    summary: forecast.abbreviatedForecast[0].textSummary[0] + '.',
                     temperature: forecast.temperatures[0].textSummary[0],
                     pop: pop && pop.length > 0 ? 'POP ' + pop + forecast.abbreviatedForecast[0].pop[0].$.units : '',
-                    cloudPopTextSummary: forecast.cloudPrecip[0].textSummary[0],
-                    popTextSummary: forecast.precipitation[0].textSummary[0],
+                    popSummary: forecast.precipitation[0].textSummary[0],
+                    cloudPopSummary: forecast.cloudPrecip[0].textSummary[0],
                     relativeHumidityPercent: forecast.relativeHumidity[0]._,
-                    windTextSummary: (forecast.winds[0].hasOwnProperty('textSummary')) ? forecast.winds[0].textSummary[0] : '',
-                    weatherTextSummary: forecast.textSummary[0]
+                    windSummary: (forecast.winds[0].hasOwnProperty('textSummary')) ? forecast.winds[0].textSummary[0] : '',
+                    windChillSummary: forecast.windChill[0].textSummary[0],
```

Comment:
Thanks for the updates. One issue: apparently *windChill* value is not always in the dataset and in those cases this line will break.